packages/vc-handler-ld-local/src/ld-credential-module.ts

```diff
@@ -91,6 +106,13 @@ export class LdCredentialModule {
 
     suite.preSigningPresModification(presentation)
 
+    if (key.type === 'Bls12381G2') {
+      return await jsonld.sign(presentation, {
```

Comment:
vc-js signPresentation has some additional logic. Which means we are not symetric in our handling of BLS signatures. https://github.com/digitalcredentials/vc-js/blob/9e40e67a3060289c899890c2cfb76681cb893368/lib/vc.js#L354

Reply:
Cannot use it because vc-js uses different packages

require('@digitalcredentials/jsonld');
const jsigs = require('@digitalcredentials/jsonld-signatures');